siriuspy/siriuspy/search/rabpm_search.py

```diff
@@ -0,0 +1,130 @@
+"""Load and process RaBPM data from static table in remote server."""
+
+from threading import Lock as _Lock
+from siriuspy.namesys import Filter as _Filter, SiriusPVName as _PVName
+import siriuspy.clientweb as _web
+
+_timeout = 1.0
+
+
+class RABPMSearch:
+    """Class with mapping BPMS in uTCA crates
```

Comment:
"...mapping devices in the uTCA crates..."
The class also lists Timing and FOFB boards.